### PR TITLE
Fix failing tests

### LINKED ISSUE #20: Add `testUpdatingMemberBeforeMinimumDelayFails`
We should add specific tests `testUpdatingMemberBeforeMinimumDelayFails` that check that updating a member to `< now + minimumDelay` reverts.

_Originally posted by @offerijns in https://github.com/centrifuge/connectors/pull/19#discussion_r928097596_

### LINKED ISSUE #21: Fix failing tests
Fix all failing tests

### PR DESCRIPTION
**Pull request by AStox**
Closes #20 
Closes #21

### CODE REVIEW

File: test/Connector.t.sol [hunk 1]
```diff
@@ -79,7 +79,7 @@ contract ConnectorTest is Test {
     }
 
     function testUpdatingMemberWorks(uint64 poolId, bytes16 trancheId, address user, uint256 validUntil) public {
-        vm.assume(validUntil > block.timestamp);
+        vm.assume(validUntil > safeAdd(block.timestamp, new Memberlist().minimumDelay()));
```
Comment (offerijns) [inline test/Connector.t.sol:82]: We should add specific tests `testUpdatingMemberBeforeMinimumDelayFails` that check that updating a member to `< now + minimumDelay` reverts.

File: test/Connector.t.sol [hunk 2]
```diff
@@ -50,8 +50,8 @@ contract ConnectorTest is Test {
         assertTrue(token_ != address(0));
 
         RestrictedTokenLike token = RestrictedTokenLike(token_);
-        assertEq(token.name(), tokenName);
-        assertEq(token.symbol(), tokenSymbol);
+        assertEq(token.name(), bytes32ToString(stringToBytes32(tokenName)));
```
Comment (offerijns) [inline test/Connector.t.sol:53]: Can you add a comment explaining why this is necessary?

Comment (AStox) [reply test/Connector.t.sol:53]: Sure! Actually I spent a lot of time thinking about this one... I ended up on the fact that probably the current behaviour is correct (where a string of "\0\0\0\0" gets treated as an empty string), so the problem then was that the test didn't reflect that. Do you agree?

I'll add a comment explaining whats going on here though

File: test/Connector.t.sol (hunk 1 above)
Comment (AStox) [reply test/Connector.t.sol:82]: Good idea

Comment (offerijns) [reply test/Connector.t.sol:85]: https://github.com/centrifuge/connectors/issues/20

**Review by offerijns — APPROVED** [review]

**Review by offerijns — APPROVED** [review]

**Pull request merged**